contents/docs/webhooks/index.md

```diff
@@ -9,57 +9,27 @@ Webhooks enable you to receive messages whenever any of your [actions](/docs/use
 
 ## Message formatting
 
-Message tokens are in the format `[type.property]`.
+By default, the webhook message format is:
 
-By default, the message format is:
 ```
-[action.name] was triggered by [user.name].
+[action.name] was triggered by [person]
 ```
 
-You can customize the message when creating or editing actions.
+The parts in brackets are called _message tokens_. You can use tokens to customize your message based on the event that triggered the webhook. Three types of tokens are available:
 
-## Token types
+### Event tokens
 
-### User
+- `[event]`: Type of the event (a standard one, such as `$pageview`, or custom, such as `completed level`). This token is formatted as a link to the event that triggered the webhook in PostHog.
+- `[event.event]`: Same as `[event]` except not formatted as a link.
+- `[event.uuid]`: ID of the event. Always in UUID format.
+- `[event.distinct_id]`: Person distinct ID associated with the event.
```

Comment:
This would not quite be the correct way to put it, because it makes it sound like like that person has a single distinct ID. They likely have multiple distinct IDs in reality, and this token just represents the one associated with the event in question.